Assets/Scripts/Player Folder/Player.cs

```diff
@@ -142,33 +150,12 @@ private void Attack()
         var (attacked, counter) = jab.Attack();
 
         if (!attacked) return;
-
-        switch (counter)
-        {
-            case 0:
-                break;
-            case 1:
-                attackParticle.Play();
-                animator.ResetTrigger("Attack Finish");
-                animator.SetTrigger("Attack Start");
-                PlayRandomAttackSound();
-                break;
-            case 2:
-                //throw new NotImplementedException("No second attack animation yet");
-                animator.ResetTrigger("Attack Finish");
-                animator.SetTrigger("Attack Start2");
-                PlayRandomAttackSound();
-                break;
-            case 3:
-                //throw new NotImplementedException("No slow attack animation yet");
-                animator.ResetTrigger("Attack Finish");
-                animator.SetTrigger("Attack Start3");
-                PlayRandomAttackSound();
-                break;
-            default:
-                Debug.LogError("HOW");
-                break;
-        }
+        
+        if (counter == 1) attackParticle.Play();
+        var variation = counter == 1 ? "" : counter.ToString();
+        animator.ResetTrigger($"Attack Finish{variation}");
+        animator.SetTrigger($"Attack Start{variation}");
+        PlayRandomAttackSound();
```

Comment:
@LauSnchzz lol look at the old version (that was probs urs idk forgot)

Reply:
yeah that aint me

Reply:
nvm it was me lmao